packages/node_modules/pouchdb-replication/src/replicate.js

```diff
@@ -131,6 +136,22 @@ function replicate(src, target, opts, returnValue, result) {
       returnValue.emit('change', outResult);
     }
     writingCheckpoint = true;
+
+    var task = src.activeTasks.get(taskId);
+    if (task) {
+      src.info().then(function (info) {
```

Comment:
might there a different way of updating `total_items` here for continuous replication other than regularly fetching the database info? we've had a bug where this callback ran into an error due to `currentBatch` being set to `undefined`—it was clearly defined before `src.info()` was called, so this happened in the meantime.